writeimage.sh

```diff
@@ -152,9 +184,22 @@ if [ -n "$IP" ] && [ -n "$GW" ] && [ -n "$DNS" ]; then
     echo "static_dns=\"$DNS\"" >> $conf
 fi
 
-msg "unmounting sdcard"
-sync
-umount $BOOT
-rmdir $BOOT
+if [ `uname` == "Darwin" ]; then
+	# copy created files over to the disk
+	DISK_PARTITION=`diskutil info $BOOT_DEV | grep "Mount Point:" | sed 's/^[^/]*//' | sed 's/ /\\ /g'`
+
+	echo "moving $BOOT/* to $DISK_PARTITION"
+
+	mv -v "$BOOT"/* "$DISK_PARTITION"
```

Comment:
Why do we need to copy all these files over to $DISK_PARTITION instead of using the $BOOT mount point, as we do for Linux?